webroot/src/components/Lists/Pagination/Pagination.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('Pagination component', async () => {
         expect(pages).to.have.length(2);
         expect(pages[0]).to.equal(1);
         expect(pages[1]).to.equal(2);
-        expect(wrapper.findComponent(LeftCaretIcon).exists(), 'previous arrow').to.equal(true);
+        // expect(wrapper.findComponent(LeftCaretIcon).exists(), 'previous arrow').to.equal(true); // Server not yet supporting this
```

Comment:
I'll remove this line.

As for when this will be implemented, I'm not sure the backend & UI designs are 100% sync-ed up, so I created #185 to track.